pyproject.toml

```diff
@@ -9,7 +9,7 @@ packages = [
     {include = "**/*.py", from = "src"},
 ]
 readme = "README.md"
-version = "0.8.0"
+version = "0.9.0"
```

Comment:
Arguably this isn't breaking because we're just adding new fields not removing the old ones.  But semantically it's a pretty big change.